web/src/styles/scss/layout.scss

```diff
@@ -42,16 +42,19 @@ footer {
 }
 
 .skip-nav {
-  background: white;
+  background: #112e51;
   height: 3rem;
   line-height: 3rem;
-  padding: 0 0.5rem;
+  padding: 1rem;
+  width: fit-content;
```

Comment:
I'm not sure if this would have an impact but IE doesn't support fit-content

Reply:
it doesn't really work anyway